kubernetes-itests/src/test/java/io/fabric8/kubernetes/ConfigMapIT.java

```diff
@@ -89,10 +95,17 @@ public void list() {
 
   @Test
   public void update() {
+    Ready<ConfigMap> configMap1Ready = new Ready<ConfigMap>(ConfigMap.class, client, "configmap1", currentNamespace);
```

Comment:
Explicit type argument can be replaced with <>

Reply:
noted. thanks!